The ordering to the tests was non-deterministic without sorting the list.
This will be fed back to the community.

--- git-1.7.9.2/t/Makefile_orig	2015-03-26 14:29:33.733092028 -0700
+++ git-1.7.9.2/t/Makefile	2015-03-26 14:29:53.247281197 -0700
@@ -17,7 +17,7 @@
 # Shell quote;
 SHELL_PATH_SQ = $(subst ','\'',$(SHELL_PATH))
 
-T = $(wildcard t[0-9][0-9][0-9][0-9]-*.sh)
+T = $(sort $(wildcard t[0-9][0-9][0-9][0-9]-*.sh))
 TSVN = $(wildcard t91[0-9][0-9]-*.sh)
 TGITWEB = $(wildcard t95[0-9][0-9]-*.sh)
 
